Podfile.lock

```diff
@@ -39,4 +39,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 39a3c51cfc7011b61bf96a8c38053b00ef42c157
 
-COCOAPODS: 1.5.2
+COCOAPODS: 1.5.3
```

Comment:
it seems you use your own pod binary.
pod is versionning in the Gemfile (currently 1.5.2), so can you update the cocopad gem in your PR ?
`bundle update cocoapods`

Reply:
Done. https://github.com/noreasonprojects/ModernAVPlayer/pull/57/commits/e3e0de38464f42d4fa3ca11137913670f8de2551